community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -96,6 +96,13 @@
             + "If true, then non-conformance will result in an error, otherwise only a warning is generated." )
     public static final Setting<Boolean> cypher_hints_error = setting( "dbms.cypher.hints.error", BOOLEAN, FALSE );
 
+    @Description( "If set to true it will disable the shortest path fallback. That means that no full path enumeration" +
+                  "will be performed in case shortest path algorithms cannot be used. This might happen in case of" +
+                  "existential predicates on the path, e.g., when searching for the shortest path containing a node" +
+                  "with property 'name=Emil'. The problem is that graph algorithms work only on universal predicates," +
+                  "e.g., when searching for the shortest where all nodes have label 'Person'." )
```

Comment:
Perhaps worth mentioning that with this setting an error could be thrown at runtime?